ide/provisioning/install-debian-packages.sh

```diff
@@ -2,8 +2,12 @@
 
 set -e
 
+# To prevent apt from displaying dialogs
+DEBIAN_FRONTEND=noninteractive
+APTOPTS="--no-install-recommends --assume-yes"
+
 echo Installing base dependencies...
-apt-get --assume-yes install make \
+apt-get install $APTOPTS make \
```

Comment:
Your commit not only adds `--no-install-recommends`, but also refactors the existing `--assume-yes` into a global variable. This makes the commit obscure.

Reply:
Moved into a separate commit.